enrolments/tests/test_validators.py

```diff
@@ -1,10 +1,75 @@
 from django.test import TestCase
-from registration.models import Student
-from enrolments.validators import validate_attendance, validate_schema
+from registration.models import Student, Family
+from accounts.models import User
+from enrolments.models import Enrolment, Class, Session
+from enrolments.validators import (
+    validate_attendance,
+    validate_schema,
+    validate_class_in_session,
+    validate_enrolment,
+)
 from django.core.exceptions import ValidationError
+from unittest.mock import patch
 
 
-class ValidatorsTestCase(TestCase):
+class EnrolmentValidatorTestCase(TestCase):
+    def setUp(self):
+        self.user = User.objects.create(email="test@staff.com")
+        self.family = Family.objects.create(
+            email="fam1@test.com",
+            phone_number="123456789",
+            address="1 Fam Ave",
+            preferred_comms="email",
+        )
+        self.session = Session.objects.create(season=Session.FALL, year="2020")
+        self.class_in_session = Class.objects.create(
+            name="Class in session",
+            session=self.session,
+            facilitator=self.user,
+            attendance={},
+        )
+
+    def test_validate_classes_in_session(self):
+        class_no_sessions = Class.objects.create(
+            name="Class with no sessions",
+            session=None,
+            facilitator=self.user,
+            attendance={},
+        )
+        self.assertIsNone(
+            validate_class_in_session(self.class_in_session, self.session)
+        )
+        self.assertRaises(
```

Comment:
might want to also test a class with a different session in addition to a class without a session

Reply:
added :)